src/encoded/static/components/facets/supressed.js

```diff
@@ -0,0 +1,9 @@
+import FacetRegistry from './registry';
+
+
+/**
+ * Used for all facets that need suppression unconditionally.
+ */
+const SuppressedFacet = () => null;
```

Comment:
This looks like a code smell. There is not a lot of utility in just returning null; or maybe I am missing the bigger picture.

Reply:
That's how you prevent a facet from appearing under all circumstances. The new /cart-<something/ endpoints return a cart facet that we never want to see.